### ISSUE TITLE
bind to=127.0.0.1 does not seem to work

### ISSUE THREAD

**Issue by Sec42** [mickem/nscp#615]
### Issue and Steps to Reproduce

I set "bind to=127.0.0.1" in nsclient.ini

### Expected Behavior

nsclient service to listen on localhost only.

### Actual Behavior

nsclient listens on 0.0.0.0

```
PS C:\Program Files\NSClient++> netstat /na |Select-String 8443

  TCP    0.0.0.0:8443           0.0.0.0:0              LISTENING
```

### Details

* NSClient++ version: 0.5.0.62

### Additional Details

```
PS C:\Program Files\NSClient++> .\nscp.exe settings --list
/modules.CheckDisk=1
/modules.CheckEventLog=1
/modules.CheckExternalScripts=1
/modules.CheckHelpers=1
/modules.CheckNSCP=1
/modules.CheckSystem=1
/modules.WEBServer=enabled
/settings/WEB/server.password=XXX
/settings/WEB/server.port=8443
/settings/default.allowed hosts=127.0.0.1,::1
/settings/default.bind to=127.0.0.1
/settings/default.password=XXX
/settings/external scripts/wrappings.bat=scripts\\%SCRIPT% %ARGS%
/settings/external scripts/wrappings.ps1=cmd /c echo If (-Not (Test-Path "scripts\%SCRIPT%") ) { Write-Host "UNKNOWN: Sc
ript `"%SCRIPT%`" not found."; exit(3) }; scripts\%SCRIPT% $ARGS$; exit($lastexitcode) | powershell.exe /noprofile -comm
and -
/settings/external scripts/wrappings.vbs=cscript.exe //T:30 //NoLogo scripts\\lib\\wrapper.vbs %SCRIPT% %ARGS%

```

**Comment by mintsoft** [comment]
@Sec42 Have you tried 0.5.2.41? (https://github.com/mickem/nscp/releases/tag/0.5.2.41) Loads of changes between 0.5.0.x and the 0.5.2 branch

**Comment by Sec42** [comment]
Yes, I tried it, and it did not seem to make any difference

**Comment by mickem** [comment]
The web interface just recently gained support for binding (previously it was only for other protocols).
So this should be fixed in 0.5.3, if not please reopen this issue
